### DISCUSSION TITLE
Talos and Longhorn 1.7.x: NFSClientInstalled is false

### DISCUSSION THREAD

**Discussion by docbobo** [longhorn/longhorn#9300] [General]
I've been running Longhorn 1.6.x on top of Talos successfully for quite a while. After updating to Longhorn 1.7.x, I noticed that the Node UI is seems to be indicating that `NFSClientInstalled` and `RequriedPackages` are false. Not sure if this is going to affect operations - but did anybody else notice this?

**Comment by derekbit** [comment]
@mantissahz Have we addressed conditions for Talos?

**Comment by innobead** [comment]
cc @c3y1huang

**Comment by mantissahz** [comment]
Talos Linux lacks some binary commands and config files like "/boot/config-" when the node controller tries to check the conditions. Therefore these two conditions will be false.

> Not sure if this is going to affect operations - but did anybody else notice this?

@docbobo conditions will not affect operations, and they just notice users that something is missing.

**Comment by derekbit** [comment]
Thanks @mantissahz. We need to clearly mention it in the condition error message.

**Comment by docbobo** [comment]
Thanks for the confirmation
